plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/config/Config.java

```diff
@@ -2,94 +2,156 @@
 
 import io.jenkins.tools.pluginmanager.impl.Plugin;
 import java.io.File;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 
-
+/**
+ * Configuration for the plugin installation manager tool.
+ * <br/>
+ * Construct it with
+ * {@code
+ * Config.builder()
+ * ...
+ * build()
+ * }
+ * <br/>
+ * Defaults for update centers will be set for you
+ */
 public class Config {
     private File pluginDir;
-    private File pluginTxt;
     private boolean showWarnings;
```

Comment:
@stopalopa this field doesn't seem to be in use, is it for later or a mistake? the getter is never called.

Reply:
I think it's fine to remove.